auth/services/oauth/relying_party.go

```diff
@@ -197,11 +197,11 @@ func determineFormat(formats map[string]map[string][]string) (string, error) {
 	for format := range formats {
 		switch format {
 		case openid4vc.VerifiablePresentationJWTFormat:
-			fallthrough
+			return format, nil
 		case openid4vc.VerifiablePresentationJSONLDFormat:
 			fallthrough
 		case "jwt_vp_json":
-			return format, nil
+			return openid4vc.VerifiablePresentationJSONLDFormat, nil
```

Comment:
this looks weird? `jwt_vp_json`is requested, but we return JSON-LD?

Note that go-did has constants for it now (`JWTPresentationProofFormat` and friends)

Reply:
I meant the other one ;)